src/modules/dashboard/components/ColonyDecisions/DecisionPreview.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import Heading from '~core/Heading';
+import Button from '~core/Button';
+import Tag from '~core/Tag';
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import { useUser, useLoggedInUser, useColonyFromNameQuery } from '~data/index';
+
+import DetailsWidget from './DetailsWidget/DetailsWidget';
+
+import styles from './DecisionPreview.css';
+
+const MSG = defineMessages({
+  type: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.type',
+    defaultMessage: `Type`,
+  },
+  team: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.team',
+    defaultMessage: `Team`,
+  },
+  author: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.author',
+    defaultMessage: `Author`,
+  },
+  edit: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.edit',
+    defaultMessage: `Edit`,
+  },
+  publish: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.publish',
+    defaultMessage: `Publish`,
+  },
+  // this is mock data for the decision title until a later PR is created to retrieve this from local storage
+  title: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.title',
+    defaultMessage: `Should we build a Discord Bot?`,
+  },
+  // this is mock data for the decision description until a later PR is created to retrieve this from local storage
+  description: {
+    id: 'dashboard.ColonyDecisions.DecisionPreview.description',
+    defaultMessage: `I think we should build a Discord bot that integrates with the Dapp and provides our community with greater transperency and also provides more convienience for us to be notified of things happening in our Colony.`,
+  },
+  // this is a placeholder for the decision type until we have the data
```

Comment:
So does this mean its fine to leave the decision type string hardcoded in the message descriptor?

Reply:
Potentially, yes. Maybe we should have a variable name or an enum with a string (depends on how it will work with decision being added to the maps for details widget and also for the future DecisionPage). I need to see how it's all coming together to understand it better... You can DM me after you implement DetailsWidget from the ActionsPage if it's not clear.